sofa-boot-project/sofa-boot-core/runtime-sofa-boot/src/main/java/com/alipay/sofa/runtime/component/impl/ComponentManagerImpl.java

```diff
@@ -86,7 +88,23 @@ public int size() {
     @Override
     public void shutdown() {
         List<ComponentInfo> elems = new ArrayList<>(registry.values());
+        // shutdown spring contexts first
```

Comment:
we should shutdown spring context after other component unregistering because ServiceComponents are beans ultimately.

Reply:
done